CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 ### JavaScript APIs
 ### Linter
 
+#### Rule status
```

Comment:
There's no contribution guideline for this headline. What does that mean to the user? Fix, feature, breaking change?